src/app/page.tsx

```diff
@@ -1,13 +1,31 @@
-import styles from "./page.module.css";
 import Layout from "./layout";
+import React from "react";
+import styles from "./page.module.css";
+import List from "../components/List/list";
 
+async function getData() {
+  const result = await fetch(
+    "https://65c53ee5dae2304e92e41ae7.mockapi.io/api/todos/"
+  );
+  // The return value is *not* serialized
+  // You can return Date, Map, Set, etc.
 
-export default function Home() {
+  if (!result.ok) {
+    // This will activate the closest `error.js` Error Boundary
+    throw new Error("Failed to fetch data");
+  }
+
+  return result.json();
+}
+export default async function Home() {
+  const data = await getData();
+  console.log(data);
   return (
     <Layout>
```

Comment:
This Layout is not needed because `layout.tsx` will automatically be applied to every route